gems/smithy/lib/smithy/vise/operations_parser.rb

```diff
@@ -4,10 +4,6 @@ module Smithy
   module Vise
     # @api private
     class OperationsParser
```

Comment:
What about having these methods public to access? (unknown if we need them later)
* a method to retrieve just operations defined on the service shape's operations (I guess technically you can retrieve this from the service shape)
* a method to retrieve just operations defined on the resource shapes